minio_service/minio_client.py

```diff
@@ -0,0 +1,80 @@
+from config import *
+from minio import Minio
+from minio.error import S3Error
+
+class MinioInit:
+    @staticmethod
+    def minio_auth(access=ConfMinio.access, secret=ConfMinio.secret):
+        return Minio(
+            'localhost:9000',
+            access_key=access,
+            secret_key=secret,
+            secure=False
+        )
```

Comment:
just keep it as function, you don't need a class for this

Reply:
removed class